server/services/personService.ts

```diff
@@ -81,6 +81,29 @@ export default class PersonService {
     }
   }
 
+  async getOffenderSentenceAndOffences(
+    token: Express.User['token'],
```

Comment:
This should be take username, rather than user token and pass it to the generated system token - see other functions here for how to generate this

Reply:
Just realised this isn't a change introduced in this PR, but it's something we do need to fix.

Reply:
#350 opened to deal with this